Makefile

```diff
@@ -265,6 +265,7 @@ test-cmd-debug: ## Run odo debug command tests
 .PHONY: test-integration
 test-integration: ## Run command's integration tests irrespective of service catalog status in the cluster.
 	$(RUN_GINKGO) $(GINKGO_FLAGS) tests/integration/
+	$(RUN_GINKGO) $(GINKGO_FLAGS) tests/integration/interactive/
```

Comment:
I'm not sure it is useful to create a new tests suite, as some commands will be a mix of interactive and non interactive mode, and the interactive mode does not need a specific environment.

Reply:
Yes, I created it for now to test it, It was intentional, I will remove it once we are confirmed the structure for test.

Reply:
As discussed, because interactive tests will work only on Linux, please make a `make test-integration-interactive`, so we can run it separately

Reply:
Out of curiosity - why can't it be run on Windows or macOS?